server/src/testFixtures/java/io/crate/expression/scalar/ScalarTestCase.java

```diff
@@ -23,7 +23,6 @@
 
 import static io.crate.testing.Asserts.assertThat;
 import static io.crate.testing.Asserts.isNull;
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I am not sure how and why I removed this. It looks as if the io.crate `assertThat` extends the assertj version and thus the seconds `import static` is not necessary.